nimbleclient/v1/restclient.py

```diff
@@ -98,7 +102,7 @@ def __init__(self, hostname, username, password, port=5392):
             self.connected = self._connect()
 
     def _connect(self):
-        """Perform NimOS authentication and session token retrieval"""
+        """Perform NimOS authentication and session token retrieval."""
```

Comment:
Let's remove the `logging.debug(sessiondata)` output from lines 117 and 123

Reply:
Agreed, I will remove this stmt